Documentation/event-reference.md

```diff
@@ -113,57 +164,62 @@ This event is a request to set the Edge Network location hint used by the Edge N
 
 -----
 
-### Edge consent response content
+## Event flags handled by Edge
 
-This event contains the latest consent preferences synced with the SDK. The Edge Network extension reads the current data collection consent settings stored in the `collect` property and adjusts its internal queueing behavior based on the value as follows:
+The following event flags are handled by the Edge extension client-side.
 
-| Value | Description | Behavior |
-| ----- | ----------- | -------- |
-| `y` | Yes | Hits are sent |
-| `n` | No | Hits are dropped and not sent |
-| `p` | Pending | Hits are queued until `y`/`n` is set; when set, queued events follow the value's behavior |
+### Request send completion
 
-#### Event dispatched by<!-- omit in toc -->
-* [`Consent.update(with:)`](https://developer.adobe.com/client-sdks/documentation/consent-for-edge-network/api-reference/#updateconsents)
+The `sendCompletion` flag is used within the Edge request to indicate whether a "complete" event should be dispatched once the streaming connection is closed. This flag is part of the `request` object, which is at the same hierarchical level as the `data` and `xdm` objects in the Edge Experience Event.
+
+```json
+{
+   "xdm": { ... },
+   "request": { "sendCompletion" : true }
+}
+```
+
+When the `sendCompletion` flag in the request is set to `true`, the Edge extension will dispatch a *paired* response event. This event notifies the caller that the connection has been closed and that no further streaming response handles will be dispatched. The [response event](#edge-content-complete) includes only the `requestId` in its data, for debugging purposes. The `parentID` of this event corresponds to the UUID of the originating request event.
```

Comment:
I like this write up, however these parameters are only send in the request content events. Having a separate flags section seems to imply that any of the request event could contain the `sendCompletion` flag. In the Android PR, I've split this information to include the `request` parameter in the request content section, while adding the `sendCompletion` in the content complete section. Having all this under the request content section is a good option too. What do you think?